runtime-light/stdlib/job-worker/job-worker-api.cpp

```diff
@@ -0,0 +1,153 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "runtime-light/stdlib/job-worker/job-worker-api.h"
+
+#include <algorithm>
+#include <chrono>
+#include <cstddef>
+#include <cstdint>
+
+#include "runtime-core/runtime-core.h"
+#include "runtime-core/utils/kphp-assert-core.h"
+#include "runtime-light/component/component.h"
+#include "runtime-light/coroutine/awaitable.h"
+#include "runtime-light/coroutine/task.h"
+#include "runtime-light/header.h"
+#include "runtime-light/stdlib/component/component-api.h"
+#include "runtime-light/stdlib/fork/fork-context.h"
+#include "runtime-light/stdlib/job-worker/job-worker-context.h"
+#include "runtime-light/stdlib/job-worker/job-worker.h"
+#include "runtime-light/streams/streams.h"
+#include "runtime-light/tl/tl-core.h"
+#include "runtime-light/tl/tl-functions.h"
+#include "runtime-light/tl/tl-types.h"
+#include "runtime-light/utils/context.h"
+
+namespace {
+
+constexpr const char *JOB_WORKER_COMPONENT_NAME = "_self";
+
+constexpr double MIN_TIMEOUT_S = 0.05;
```

Comment:
`inline` like in case in fork-contex.h?

Reply:
`inline` is useless in `*.cpp` files since they are never included.